drivers/adc/ad7768/ad7768.c

```diff
@@ -714,6 +715,9 @@ int32_t ad7768_setup_finish(ad7768_dev *dev,
 
 	ret |= no_os_gpio_direction_output(dev->gpio_reset, dev->gpio_reset_value);
 
+	/* Delay to ensure device is out of reset */
+	no_os_mdelay(2);
```

Comment:
this might potentially break projects that use this driver but don't have no_os_delay in their src.mk, let's see the CI